pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -103,11 +104,13 @@ public static Object jsonPath(Object object, String jsonPath) {
 
   @ScalarFunction
   public static Object[] jsonPathArrayDefaultEmpty(Object object, String jsonPath) {
-    try {
-      return jsonPathArray(object, jsonPath);
-    } catch (Exception e) {
-      return new Object[0];
+    final Object value;
```

Comment:
(minor) After fixing `jsonPathArray()`, this method can be simplified by checking the return value of `jsonPathArray()`